tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/accessibility.py

```diff
@@ -82,10 +86,16 @@ def a11y_toggle_switch(self, switch):
             [switch])
 
     def a11y_toggle_filters(self):
-        self.a11y_toggle_switch(self.marionette.find_element(*self._filter_enable_switch_locator))
+        Wait(self.marionette).until(
+            expected.element_displayed(*self._filter_enable_switch_locator))
+        self.marionette.find_element(*self._filter_enable_switch_locator).tap()
```

Comment:
I don't think you can replace the a11y_toggle_switch function with what you're doing there.
I think this was done for a reason, see here what the function is doing:
http://mxr.mozilla.org/gaia/source/tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/accessibility.py#79